Zion's End/src/Grid.cpp

```diff
@@ -149,6 +149,21 @@ void Grid::Render(sf::RenderTarget& renderer)
 	}
 }
 
+#pragma warning(push)
+#pragma warning(disable: 4715)
+sf::Vector2f Grid::GetBiblePosition()
```

Comment:
There are two way out of this:
- Assume that we are the only authors of the maps, and that we aren't trying to crash it on purpose
- Create a map validator, that would check for Starting and End Tile presence.